lib/msf/core/opt.rb

```diff
@@ -62,6 +62,10 @@ def self.SSLVersion
       )
     end
 
+    def self.RHOST_URL(default=nil, required=false, desc="The target URL, only applicable if there is a single URL")
```

Comment:
```suggestion
    def self.RHOST_URL(default = nil, required = false, desc = 'The target URL, only applicable if there is a single URL')
```

https://github.com/rubocop-hq/ruby-style-guide#spaces-around-equals